tests/unit/test_api_independence.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
could this be a pre-commit which looks for `import llama_stack$` or `from llama_stack `? would it be simpler?